db/migrate/20230317132501_create_free_trainings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFreeTrainings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :free_trainings do |t|
+      t.string :bodyPart
```

Comment:
It is very important to follow the ruby naming convention that is `snake_case` instead `camelCase`.

The js convention is `camelCase`. Each language has its singularities and it is essential to follow that.